playground/pages/[...slug].vue

```diff
@@ -0,0 +1,20 @@
+<script setup>
+const route = useRoute();
+
+const story = ref();
+try {
+  story.value = await useAsyncStoryblok(route.path, {
```

Comment:
We are planning a new refactor of the playground and this could be a starting point, thanks!

Reply:
I would love to help with that refactor! 🥳

Reply:
I have only made one node package from the ground up for nuxt where I tried to stick to the recommended stuff from nuxt.
https://github.com/SebbeJohansson/sst-sdk-nuxt

I think moving to that structure would make sense (how to refer to the package etc).

Reply:
Oh, but with the refactor, I mean, it was more concerning the use cases from the Storyblok test space we have for the SDKs, creating different types of blocks, languages, and so on.

For the thing you mentioned, it's because we are using `semantic-release`, and is a setup for the whole SDK ecosystem, @alexjoverm can tell you more about it, is the guru of versioning stuff 🚀

Reply:
Ah okay interesting! I'll have to read some on that! Thanks. And would love to hear more about it from @alexjoverm 😁